src/odemis/dataio/tiff.py

```diff
@@ -79,12 +79,28 @@
 # TODO: make sure that _all_ the metadata is saved, either in TIFF tags, OME-TIFF,
 # or in a separate mechanism.
 
-# Add special fei_md tag to the tag "library"
+# Add special tags to the tag "library"
 TIFFTAG_FEI_MD = 34682
 TIFFTAG_FEI_MD_XML = 34683
+TIFFTAG_ZEISS_MD_1 = 34118
+TIFFTAG_ZEISS_MD_2 = 34119
 tag_fei = T.TIFFFieldInfo(TIFFTAG_FEI_MD, -1, -1, T.TIFFDataType.TIFF_ASCII, T.FIELD_CUSTOM, True, False, b"fei_md")
 tag_fei_xml = T.TIFFFieldInfo(TIFFTAG_FEI_MD_XML, -1, -1, T.TIFFDataType.TIFF_ASCII, T.FIELD_CUSTOM, True, False, b"fei_md_XML")
-ext = T.add_tags([tag_fei, tag_fei_xml]) # Note: if you ever dereference ext, libtiff will crash
+tag_zeiss_md_1 = T.TIFFFieldInfo(TIFFTAG_ZEISS_MD_1, -3, -3, T.TIFFDataType.TIFF_BYTE, T.FIELD_CUSTOM, False, True, b"zeiss_md_1")
+tag_zeiss_md_2 = T.TIFFFieldInfo(TIFFTAG_ZEISS_MD_2, -3, -3, T.TIFFDataType.TIFF_BYTE, T.FIELD_CUSTOM, False, True, b"zeiss_md_2")
+ext = T.add_tags([tag_fei, tag_fei_xml, tag_zeiss_md_1, tag_zeiss_md_2]) # Note: if you ever dereference ext, libtiff will crash
+
+# Hack: they are officially defined as "ASCII" in the file, but they are not
+# null terminated. So read them "raw", clip to the count, and then convert from bytes to the right string format.
+# However, for now pylibtiff doesn't support reading variable length tags via add_tags(),
+# So we have to explicitly change the format to indicate it'll be variable (and GetField will work fine)
+T.tifftags[TIFFTAG_ZEISS_MD_1] = ((T.ctypes.c_uint32, T.ctypes.c_char), lambda d: d[1][:d[0]])
+T.tifftags[TIFFTAG_ZEISS_MD_2] = ((T.ctypes.c_uint32, T.ctypes.c_char), lambda d: d[1][:d[0]])
+
+# suppress warnings and errors from libtiff (written to stderr by default)
+T.suppress_errors()
```

Comment:
Let's keep the error active for now (you can just leave the line, but comment it out). So far, the annoying ones were the warnings. It can still be handy to see an error if there is one.